### PR TITLE
feat: og images

### PR DESCRIPTION
**Pull request by Julien-R44**
Follow up of #89 because the diff was totally broken

- I've added `summary` to all our pages. Some of them probably need some changes, but I think it's a good start
- I've generated all the images and pushed them.
- I've lengthened the descriptions on the OG images as requested in the other PR

PR is normally ready to be merged

### CODE REVIEW

File: bin/build.ts [hunk 1]
```diff
@@ -30,6 +36,68 @@ const IMPORTER = (filePath: string) => {
...
+        .toFile(output)
+    }
+  } catch (e) {
+    console.error('Failed to generate og image', e)
+  }
+
+  /**
+   * Insert the og:image meta tag in the output
+   */
+  const ogImageUrl = output.replace('public/', 'https://feat-og-images-v2.v6-docs.pages.dev')
```
Comment (Julien-R44) [inline bin/build.ts:95]: i've updated the base url so we can see what it looks like. we should not forget to change this before merging

Comment (thetutlage) [reply bin/build.ts]: Does the URL has to be an absolute path with the domain?. Can't we have a path with a slash `output.replace('public/', '/')`

Comment (RomainLanz) [reply bin/build.ts]: It has to be an absolute path for third-party content to read it

Comment (thetutlage) [reply bin/build.ts]: Okay.

**Pull request merged**